third_party/tinycbor/CMakeLists.txt

```diff
@@ -38,11 +38,6 @@ add_library( tinycbor
 target_include_directories( tinycbor SYSTEM
                             PUBLIC tinycbor/src )
 
-# Link math library. This is not needed on Windows.
-if( NOT ${CMAKE_SYSTEM_NAME} STREQUAL "Windows" )
-    target_link_libraries( tinycbor PRIVATE m )
-endif()
```

Comment:
If this is needed when not on Windows, shouldn't it be left in? Or is it linked elsewhere?

Reply:
oops thanks